.github/workflows/deploy.yml

```diff
@@ -0,0 +1,26 @@
+name: deploy to lambda
+on:
+  push:
+    branches:
+      - master
+jobs:
+  deploy_to_lambda:
+    name: build and deploy lambda
+    strategy:
+      matrix:
+        platform: [ubuntu-latest]
+    runs-on: ${{matrix.platform}}
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install zip
+        run: sudo apt install zip
+      - name: create zip bundle
+        run: zip -qq -r ./bundle.zip ./api-dynamodb-ingest
+      - name: default deploy
+        uses: appleboy/lambda-action@master
+        with:
+          aws_access_key_id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws_secret_access_key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
```

Comment:
are you storing keys?

Reply:
on the secrets of the github repository yes

Reply:
the key is for my user right now so if that is deactivated probably this will become invalid